### ISSUE TITLE
Assumption message for Skipped scenario

### ISSUE THREAD

**Issue by OxygeneIV** [trivago/cluecumber#299]
Hi, pretty new in the Cluecumber field...

I do get a Skipped scenario when using Assumptions and Failed scenario for Assertions, e.g:
```
Assumptions.assumeTrue(5 == 4, "Nothing available, stopping test");  => Skipped
Assertions.assertEquals(5 == 4, "Nothing available, stopping test");  => Failed
```
I do however not see the Assumption message for the skipped test which makes it hard to know the reason for Skipping.
Since we do not want a failed (red) test in these situations, it would be nice in the UI to quickly see the "assume message" along the Scenario name, as it is now for Assertions.

Perhaps there is another way to display a message for Assumed-skipped tests without having to click on the Scenario link for further details.

/M

**Comment by lunmag** [comment]
V 2.9.4, using maven

**Comment by bischoffdev** [comment]
Hello, this is the Cluecumber project, not Cucumber. So I don't think this is the right place for your question.

**Comment by bischoffdev** [comment]
...or did I misunderstand it and you are talking about the reporting in this case?
If so, please attach a minimal code example for your case.

**Comment by OxygeneIV** [comment]
Yes, the reporting:

Typical Java code: 
`Assumptions.assumeTrue(5 == 4, "No V75 available, stopping test");`

I'd like the Assume message to appear as for Assertions (at arrow position)
![image](https://user-images.githubusercontent.com/2960057/207813067-f941b14c-11dc-4f31-9744-d340174972fd.png)

**Comment by bischoffdev** [comment]
Can you attach a json file so I can check out if these messages are in there? Because if they are not, there is no way to display those.

**Comment by lunmag** [comment]
Had to rename due to extension validation...

[cucumber.json.txt](https://github.com/trivago/cluecumber/files/10235346/cucumber.json.txt)

**Comment by bischoffdev** [comment]
Perfect, thank you! Will investigate now.

**Comment by bischoffdev** [comment]
I can see the issue, fixing it now.

**Comment by bischoffdev** [comment]
![image](https://user-images.githubusercontent.com/5775857/207848902-d1505e46-3dc1-4301-a88c-ba1ac6452ef6.png)

Fixed, releasing it now for v3.0.1.

**Comment by bischoffdev** [comment]
Release done.
